test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1564,9 +1574,16 @@ def tearDown(self):
 
         for sched in self.scheds:
             self.scheds[sched].cleanup_files()
+        self.revert_server(self.server)
```

Comment:
why did you add the revert operation to tearDown() ?

Reply:
Sorry my bad, added this to debug and forgot to remove.